.github/workflows/ci.yml

```diff
@@ -193,3 +195,12 @@ jobs:
               if: startsWith(github.ref, 'refs/tags/v')
               with:
                   generate_release_notes: true
+            - name: test push image
```

Comment:
@unkcpz this should be removed right? Otherwise looks good.

Reply:
Correct! Purged.